### ISSUE TITLE
[Suggestion] EnderBattery

### ISSUE THREAD

**Issue by AceologyGaming** [Chicken-Bones/EnderStorage#87]
Kinda self-explanatory, akin to the existing tank and chest, a way to teleport RF, (seeing as tesseracts won't be back for a while, and EIO's thing is weird), possibly with an in-inventory item to charge your inventory.

**Comment by Ecu** [comment]
There are already a plethora of mods that offer teleportation of energy.  Additionally, Ender Storage was always a concise mod that could work along vanilla as well as along other mods.  Until such a time that vanilla has an energy system of its own, I do not agree with adding an energy teleportation in Ender Storage.

**Comment by drunkripper** [comment]
Yeah but they all either are buggy (FluxNetworks) or can only transmit a small amount of power.

**Comment by Ecu** [comment]
Your argument does not change my position at all.
